lineup.js

```diff
@@ -1,9 +1,14 @@
 function calculateTotalSalary(lineup) {
-  return 0
+  return lineup.reduce((total, lineup) => {
+    return total + lineup.salary
+  }, 0)
 }
 
 function getPositionCounts(lineup) {
-  return {}
+  return lineup.reduce((position, player) => {
```

Comment:
Naming note, this function is building an object that contains all of the positions from all the objects in the list and home many times they occur in the list. Naming that object `position` is confusing since that name would suggest a single position for a single player. A name like `positions` or `positionCounts` would be much better.